src/js/common/api/kontext/freqTree.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright 2018 Tomas Machalek <tomas.machalek@gmail.com>
+ * Copyright 2018 Institute of the Czech National Corpus,
+ *                Faculty of Arts, Charles University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Observable } from 'rxjs';
+import { map, flatMap, tap } from 'rxjs/operators';
+
+import { cachedAjax$ } from '../../ajax';
+import { HTTPHeaders, IAsyncKeyValueStore } from '../../types';
+import { CorpusInfoAPI, APIResponse as CorpusInfoApiResponse } from './corpusInfo';
+import { ConcApi, QuerySelector } from './concordance';
+import { ViewMode } from '../abstract/concordance';
+import { LemmaVariant } from '../../query';
+
+export enum FreqSort {
+    REL = 'rel'
+}
+
+export interface HTTPResponse {
```

Comment:
Isn't it possible to reuse an existing type from the `common/api/kontext/freqs.ts` ?